tutorials/viewports/multiple_resolutions.rst

```diff
@@ -3,21 +3,24 @@
 Multiple resolutions
 ====================
 
-Base resolution
----------------
+Base size
+---------
 
-A base screen resolution for the project can be specified in the project
-settings under ``display``, ``window``
+A base size for the project can be specified in the project settings
```

Comment:
I think "screen size" or "window size" would sound better. "size for the project" is a little too generic.